samples/net/mqtt_sn_publisher/Kconfig

```diff
@@ -0,0 +1,18 @@
+# Configuration options for MQTT-SN sample
+
+# Copyright (c) 2022 René Beckmann
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "MQTT-SN sample application"
+
+config MQTT_SN_GATEWAY_IP
+	string "IP of the MQTT-SN gateway"
+
+config MQTT_SN_GATEWAY_PORT
+	int "Port of the MQTT-SN gateway"
+
+config MQTT_SN_BUFFER_SIZE
```

Comment:
For the sample apps, I would suggest that `SAMPLE` is prefixed to config symbols so `SAMPLE_MQTT_SN_GATEWAY_IP` etc. This makes it clear that the symbol is only available for this sample. See some other network samples for examples how to do it.

Reply:
Good idea - done.